runtime/src/extensions.rs

```diff
@@ -208,7 +213,7 @@ mod tests {
 		Ok((wasm_binary, code_hash))
 	}
 
-	pub fn function_selector(name: &str) -> Vec<u8> {
+	fn function_selector(name: &str) -> Vec<u8> {
```

Comment:
There is also the [ink::selector_bytes!](https://docs.rs/ink_lang/latest/ink_lang/macro.selector_bytes.html) macro

Reply:
Wouldn't want to take a dependency on ink! in the runtime.

Reply:
Yes figured that too while I was working out :)